src/main/java/org/jabref/gui/slr/ExistingStudySearchAction.java

```diff
@@ -117,11 +125,8 @@ protected void setupRepository(Path studyRepositoryRoot) throws IOException, Git
      * @return Path of current panel database directory or the standard working directory
      */
     private Path getInitialDirectory(Path standardWorkingDirectory) {
-        if (frame.getBasePanelCount() == 0) {
-            return standardWorkingDirectory;
-        } else {
-            Optional<Path> databasePath = frame.getCurrentLibraryTab().getBibDatabaseContext().getDatabasePath();
-            return databasePath.map(Path::getParent).orElse(standardWorkingDirectory);
-        }
+        return stateManager.getActiveDatabase().map(database -> database.getDatabasePath().map(Path::getParent)
+                                                                        .orElse(standardWorkingDirectory))
+                           .orElse(standardWorkingDirectory);
```

Comment:
Doppelt gemoppelt?

Reply:
getParent is different in the first. Took it from the original by Dom

Reply:
But the orElse is duplicated. The inner one can be removed